.travis.yml

```diff
@@ -12,11 +11,13 @@ env:
   - VARNISH_VERSION=4.0
 
 matrix:
+  allow_failures:
+    # HHVM support for CURLOPT_PROTOCOLS was added recently but not yet
```

Comment:
is this a new problem? could we use some older guzzle / different psr-7 when running with hhvm?